frontend/src/model.tsx

```diff
@@ -899,24 +895,10 @@ async function GetAppConfig() {
   }
 }
 
-async function LoadCsrfToken(logger: ILogger): Promise<string | null> {
-  try {
-    const token = await getCsrfToken(axiosInstance)
-    setCsrfTokenHeader(axiosInstance, token)
-    logger.info(`Set X-CSRF-Token header to ${token}`)
-    return token
-  } catch (error) {
-    logger.error('Could not fetch CSRF token')
-    notify(i18n.t('global.errors.csrfError'), 'error')
```

Comment:
nit-pick: we can get rid of `global.errors.csrfError`

Reply:
Right!

Reply:
Removed